BINDINGS.md

```diff
@@ -125,11 +125,13 @@ Example:
 When "Properties" autocomplete contains <string> in all elements
 
 * Select value inside dropdown:
+[Given("user selected \"(.*) in (.*)")]
```

Comment:
`[Given("user selected \"(.*?)\" in (.*?)")]`

Reply:
Fixed